config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIG_NOT_COMPATIBLE:
+    title: 'The current configuration is not compatible with this version of Magento'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DEPLOY_SCD_OPTIONS_IGNORANCE:
+    title: 'Scd options ignored as deploy didn''t run on the build phase'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DEPRECATED_VARIABLES:
+    title: 'The configuration contains deprecated variables or values'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ENVIRONMENT_CONFIG_NOT_VALID:
+    title: 'Environment configuration is not valid'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIG_WRONG_JSON_FORMAT:
+    title: 'Json type configuration can''t be decoded'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SERVICE_VERSION_NOT_COMPATIBLE:
+    title: 'The current configuration is not compatible with this version of Magento'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SERVICE_PASSED_EOL:
+    title: 'Some services have passed EOL'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DEPRECATED_MYSQL_SEARCH_ENGINE:
+    title: 'The MySQL search configuration option is deprecated'
+    suggestion: 'Use Elasticsearch instead'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ENV_PHP_MISSED:
+    title: 'Magento state indicated as installed but configuration file app/etc/env.php was empty or did not exist.'
+    suggestion: 'Required data will be restored from environment configurations and from .magento.env.yaml file.'
+    stage: deploy
+    step: install-update
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SPLIT_DB_CUSTOM_CONNECTION_USED:
+    title: 'For split databases used custom connections'
+    stage: deploy
+    step: 'install-update:db-connection'
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DB_CONFIG_NOT_COMPATIBLE_WITH_SLAVE:
+    title: 'You have changed db configuration that not compatible with slave connection.'
+    stage: deploy
+    step: 'install-update:db-connection'
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SPLIT_DB_ENABLING_SKIPPED:
+    title: 'Enabling a split database will be skipped.'
+    stage: deploy
+    step: 'install-update:split-db'
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_NOT_ENOUGH_DATA_SPLIT_DB_VAR:
+    title: 'Variable SPLIT_DB does not have data which were already split.'
```

```suggestion
    title: 'The SPLIT_DB variable does not have data which were already split.'
```

Message is a bit unclear.  SPLIT_DB variable doesn't have the right configuration?